### PR TITLE
Adapt OpenTelemetry native metrics protocol

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java [hunk 1]
```diff
@@ -175,20 +173,18 @@ private static long tsToMilli(Timestamp timestamp) {
...
-            return;
-        }
+    @Override
+    public void active(
+        List<String> enabledRules,
+        MeterSystem meterSystem,
+        GRPCHandlerRegister grpcHandlerRegister)
+        throws ModuleStartException {
+
+        final List<Rule> rules = Rules.loadRules("otel-oc-rules", enabledRules);
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:183]: So, all MAL scripts are still going to be located in `otel-oc-rules`? Should we rename the folder to `otel-rules`?

Comment (kezhenxu94) [reply oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:183]: We can do that.

Comment (wu-sheng) [reply oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:183]: Let's do this, but note we need to adjust some docs about the folder name including docs and package config files.

Comment (wu-sheng) [reply oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:183]: @yswdqz Notice this. I prefer I can merge this PR first, then you should adjust your PR accordingly.

Comment (wu-sheng) [reply oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:183]: @kezhenxu94 We need a breaking change about the folder name change.

Comment (yswdqz) [reply oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java:183]: > @yswdqz Notice this. I prefer I can merge this PR first, then you should adjust your PR accordingly.

Got it.

**Review by wu-sheng — DISMISSED** [review]
LGTM

File: docs/en/setup/backend/opentelemetry-receiver.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # OpenTelemetry receiver
 
 The OpenTelemetry receiver supports ingesting agent metrics by meter-system. The OAP can load the configuration at bootstrap.
-If the new configuration is not well-formed, the OAP may fail to start up. The files are located at `$CLASSPATH/otel-<handler>-rules`.
-E.g. The `oc` handler loads rules from `$CLASSPATH/otel-oc-rules`.
+If the new configuration is not well-formed, the OAP may fail to start up. The files are located at `$CLASSPATH/otel-rules`.
```
Comment (wu-sheng) [inline docs/en/setup/backend/opentelemetry-receiver.md:4]: FYI @wankai123 Please notice this folder has been renamed.

**Comment by wu-sheng** [comment]
e2e are failing, it seems missing some adjustments.

**Comment by wu-sheng** [comment]
At least, starter/pom.xml is forgotten to change.

image

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**